src/require.jl

```diff
@@ -168,5 +168,54 @@ function __init__()
                                                 t1 - t0)
         end
     end
+    @require QuadDIRECT = "dae52e8d-d666-5120-a592-9e15c33b8d7a"
+        export QuadDIRECT
```

Comment:
why export?

Reply:
The Optim solvers are exported. So I thought I'd export this as well. We don't need to though. Would you prefer to not export this?

Reply:
if you have to use requires, then `using QuadDIRECT` is already done on the user side, so it's all already there.